### PR TITLE
📺 Add Rebase slides

### PR DESCRIPTION
**Pull request by juliaogris**
### Changes proposed in this PR
- Add `Rebase interactively` slide
- Add `Rebase a feature branch` slide
- Add `Update branch on Github` slide
- Remove `trim` from custom highlights as it messes with branch diagram

### CODE REVIEW

File: templates/slides.tmpl [hunk 1]
```diff
@@ -111,15 +111,15 @@
...
-     
+
         if (!match) {
           hljs.highlightBlock(el)
 
           return
         }
 
         languageTag = match[1]
-        el.innerText = match[2].trim()
```
Comment (pentaphobe) [inline templates/slides.tmpl:122]: Am curious what the fail condition is on this

The parsing appeared to be allowing extra whitespace in some scenarios


But ultimately: LGTM...  suspiciously

File: content/gotraining.slide [hunk 2]
```diff
@@ -339,6 +339,65 @@ Don't be afraid!
 Try it out on a new, throwaway branch.
 
 
+* Rebase interactively
+
+[[https://stackoverflow.com/a/36644561][Follow instruction]] to setup VSCode as `git` editor (iff that's your preferred editor).
```
Comment (camscale) [inline content/gotraining.slide]: s/iff/if/ - i assume you don't mean iff.

File: content/gotraining.slide [hunk 3]
```diff
@@ -639,7 +639,8 @@ If `master` on upstream has advanced in the meantime:
 - Create `Storer` interface with [[https://en.wikipedia.org/wiki/Create,_read,_update_and_delete][CRUD]] methods for `Puppy`
 - Write a `MapStore` implementation of `Storer` backed by a `map`
 - Write a `SyncStore` implementation of `Storer` backed by a [[https://golang.org/pkg/sync/#Map][sync.Map]]
-- Bonus points: Write tests against the `Storer` interface in a [[https://godoc.org/github.com/stretchr/testify/suite][suite]] and run the suite with both implementations
+- Find hints at [[https://github.com/anz-bank/go-samplerest/blob/master/pkg/pet/types.go][anz-bank/go-samplerest]]
+- Tests against the `Storer` interface and run in [[https://godoc.org/github.com/stretchr/testify/suite][suite]] with both implementations
```
Comment (camscale) [inline content/gotraining.slide]: taking away their bonus points. oh you're a hard taskmaster.

Did you mean s/^Tests/Test/ ?

**Review by camscale — DISMISSED** [review]
LGTM modulo trivia.

Although it is very disappointing that you're fixing a bug in this PR without a ticket tracking the bug. There are rules, you know, and they must be followed. Perhaps we need a pre-receive hook to stop this malarky.

File: content/gotraining.slide (hunk 2 above)
Comment (juliaogris) [reply content/gotraining.slide]: I did mean `if and only if`

Comment (juliaogris) [reply content/gotraining.slide]: I did mean `if and only if`

Comment (juliaogris) [reply content/gotraining.slide]: But now that I've read up on what it really means it makes no sense at all... I think.

Comment (juliaogris) [reply content/gotraining.slide]: But now that I've read up on what it really means it makes no sense at all... I think.

**Comment by juliaogris** [comment]
Thanks for the reviews @pentaphobe and @camh-anz - did that 🛒 finally make its way out of your wings? I actually started writing the issues up.... but great tiredness overcame me.

PTAL

**Review by camscale — APPROVED** [review]
LGTM

**Comment by camscale** [comment]
just busting your chops over the issue link. this is fine. no the :shopping_cart: did not make it's way out, and it's multiplying. a couple more have joined it for company.

**Pull request merged**